CVLByExample/summary/Library/DirectSummary/README.md

```diff
@@ -0,0 +1,28 @@
+
+# Direct VS Wildcard Summary
+This directory demonstrates using the following summary definitions:
+- direct (contract level) summary of base and overloaded functions.
+- wildcard summary of all library functions.
+
+There are several specs each summarizing different functions.
+- `AllDirect.spec` - every function being called (from solidity) is given its own direct summarization.
+   
+    Run this version via ```certoraRun certora/conf/AllDirect.conf```
+    [A report of this run](https://prover.certora.com/output/1902/1c2659cb4fd44abd82992d6585e0e345?anonymousKey=eab39d9370595a2b31c450c14d3b6ddff047c01e)
+
+     The summarization 
```

Comment:
need to rewrite this. line is too long and it is formatted badly, see how it looks in the preview/web

Reply:
This is the way to give a link. This is how it looks in the documentation 
https://github.com/Certora/Examples/tree/sitvanit/summary-examples/CVLByExample/summary/Library/DirectSummary